csrc/ops/utils.cpp

```diff
@@ -323,6 +323,13 @@ IterDomain* newOutputIterDomain(
       continue;
     }
 
+    NVF_ERROR(
```

Comment:
Was this moved due to the change of the interpretation? Or just for clarity?

Reply:
Without this change, pointwise ops like `set` don't propagate the DID on a broadcast dimension from input to output. Although `b(DID)` is semantically the same as without DID, I think it's better to get the input and the output to agree.